src/Psalm/Config.php

```diff
@@ -623,7 +623,7 @@ public static function loadFromXMLFile(string $file_path, string $current_dir):
 
         try {
             $config = self::loadFromXML($base_dir, $file_contents, $current_dir, $file_path);
-            $config->hash = sha1($file_contents . \PSALM_VERSION);
+            $config->hash = sha1($file_contents);
```

Comment:
Is there a reason for this? It seems sane to invalidate hash when Psalm is updated

Reply:
That was a bug - I initially tried to add cli overrides on this line but realized they weren't available